src/molecules/list-item/image-list-item.js

```diff
@@ -24,34 +27,43 @@ class ImageListItem extends Component {
 			text: PropTypes.string.isRequired,
 			nostyle: true
 		})
-	}
+	};
 
 	static defaultProps = {
 		buttonAttributes: {},
 		subtitle: null
-	}
+	};
 
-	renderButton = (buttonProps, text) => (
+	renderButton = (buttonProps, text) =>
 		buttonProps.onClick ? (
-			<ArrowButton {...buttonProps} mt={3}>{text}</ArrowButton>
+			<ArrowButton {...buttonProps} mt={3}>
+				{text}
+			</ArrowButton>
 		) : (
-			<ArrowButtonLink {...buttonProps} mt={3}>{text}</ArrowButtonLink>
-		)
-	)
+			<ArrowButtonLink {...buttonProps} mt={3}>
+				{text}
+			</ArrowButtonLink>
+		);
 
 	render() {
-		const { imageAttributes, buttonAttributes, title, subtitle, children, ...attrs } = this.props;
+		const { imageAttributes, buttonAttributes, title, subtitle, children, to, href, ...attrs } = this.props;
 		const { text, ...buttonProps } = buttonAttributes;
 
 		return (
 			<ListItemContainer {...attrs}>
-				<ListItemImage {...imageAttributes} />
-				<ListItemHeader mt={3}>
-					<ListItemTitle>{title}</ListItemTitle>
-					{subtitle && <Subtitle pb={1} color="#717171">{subtitle}</Subtitle>}
-				</ListItemHeader>
-				<ListItemBody>{children}</ListItemBody>
-				{text && this.renderButton(buttonProps, text)}
+				<LinkWrapper to={to} href={href}>
+					<ListItemImage {...imageAttributes} />
+					<ListItemHeader mt={3}>
+						<ListItemTitle>{title}</ListItemTitle>
+						{subtitle && (
+							<Subtitle pb={1} color="#717171">
+								{subtitle}
+							</Subtitle>
+						)}
+					</ListItemHeader>
+					<ListItemBody>{children}</ListItemBody>
+					{text && this.renderButton(buttonProps, text)}
```

Comment:
renderButton should not be inside of the link wrapper. This outputs the following html: 
```
  <a href="/something">
    <h2>title</h2>
    <a href="/something">I'm the button</a>
  </a>
```

According to the w3c spec [the anchor tag is defined as](http://www.w3.org/TR/html5/text-level-semantics.html#the-a-element): 
>  Transparent, but there must be no interactive content descendant.

Reply:
Good catch, this is fixed.